Source/WebCore/dom/Node.h

```diff
@@ -754,10 +754,10 @@ class Node : public EventTarget, public CanMakeCheckedPtr {
     const uint16_t m_typeBitFields;
     mutable OptionSet<StateFlag> m_stateFlags;
 
-    CheckedPtr<ContainerNode> m_parentNode;
+    ContainerNode* m_parentNode { nullptr };
     CheckedPtr<TreeScope> m_treeScope;
-    CheckedPtr<Node> m_previous;
-    CheckedPtr<Node> m_next;
+    Node* m_previous { nullptr };
+    Node* m_next { nullptr };
```

Comment:
It seems so scary to make these raw pointers again.

Reply:
I agree we need to revisit but this just restores shipping behavior. The issue is that using a WeakPtr here is super tricky because of DOM & CSS JIT and I am not familiar with the JIT code at all.